qutebrowser/completion/completionwidget.py

```diff
@@ -203,6 +203,10 @@ def set_model(self, model):
             sel_model.deleteLater()
         for i in range(model.rowCount()):
             self.expand(model.index(i, 0))
+
+        if model.column_widths:
```

Comment:
Is this check necessary? I can't see a case where the model would have `column_widths` set to a negative value, as it's set in `BaseCompletionModel`, right?